src/main/java/com/internet/shop/Main.java

```diff
@@ -16,5 +20,32 @@ public static void main(String[] args) {
         System.out.println(manufactureService.delete(1L));
         System.out.println(manufactureService.get(2L));
         System.out.println(manufactureService.getAll());
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driverJohn = new Driver("John", "323");
+        driverService.create(driverJohn);
+        System.out.println(driverService.getAll());
+        driverService.delete(1L);
+        Driver driverBob = new Driver("Bob", "445");
+        System.out.println(driverService.create(driverBob));
+
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        Car car = new Car("aaaa", manufactureService.get(2L));
+        carService.create(car);
+        carService.addDriverToCar(driverService.get(5L), car);
+        System.out.println(carService.getAll());
+        System.out.println(carService.getAllByDriver(1L));
+        carService.removeDriverFromCar(driverJohn, car);
+        carService.delete(1L);
+        System.out.println(carService.update(car));
+        System.out.println(carService.get(3L));
+        System.out.println(carService.getAll());
+        Car updated = carService.get(2L);
```

Comment:
```suggestion
        Car updated = carService.get(car2.getId());
```
we can use getId() method instead of hardcoding ids